proposed/psr-8-hug/psr-8-hug.md

```diff
@@ -31,7 +31,15 @@ condition that will prevent an infinite loop.  For example, an object MAY be
 configured to only allow up to 3 mutual hugs, after which it will break the hug
 chain and return.
 
-4. An object MAY take additional actions, including modifying state, when hugged.
+4. An object whose wouldHug() method is invoked MUST return a boolean representing
+its current willingness to engage in a hug with the Huggable passed as the first
+parameter. However, the return value is not a guarantee that future invocations
+will return the same result.
+
+5. An object MUST NOT take additional actions or modify state when wouldHug() is
```

Comment:
An excellent point. The original intent was to make the `wouldHug()` request "penalty-free" — a way of determining the likely response without triggering any potentially negative side effects. But it's unrealistic that the spec REQUIRE it be completely devoid of side effects. I'd support pulling 5.